server/views/accessibility-statement.html

```diff
@@ -0,0 +1,93 @@
+{% extends 'form-layout.html' %}
+
+
+{% block formContent %}
```

Comment:
I think this can use content rather than formContent as there doesn't appear to be any form fields on the page.

Reply:
Well spotted